include/simdjson/common_defs.h.in

```diff
@@ -153,10 +153,12 @@ constexpr size_t DEFAULT_MAX_DEPTH = 1024;
      * the regular visual studio or clang under visual
      * studio.
      */
```

Comment:
Moving common_defs.h is likely to break the single-header generation code (though that would be fixable, of course). Note that the files in the singleheader directory are regenerated upon release from the files in include and src, they are not maintained by hand.